recipe/meta.yaml

```diff
@@ -60,10 +60,11 @@ requirements:
     - gmp  # [unix]
     - libva  # [linux64]
     - aom
-    - svt-av1
+    - svt-av1  1.4.1   # latest version to support ffmpeg 5.0
     - ffnvcodec-headers  # [linux64 or win]
     - libopus
     - libass  # [not win]
+    - dav1d
```

Comment:
Would have presumably needed a rerender for the pins to take hold

Reply:
i guess i'm not too worried about the pinnings being exact. If it gets built twice with dav1d 1.2.0 is it that big of a deal?

Reply:
Looking at one of the linux logs, it seems that dav1d 1.0.0 was used. so that was my goal. one build with 1.0.0 before 1.2.0 migrates over.